Validator russian individual tax number

tabbols95 opened the pull request:
This Merge Request proposes the implementation of a validation feature for the Russian Individual Taxpayer Identification Number (INN). This enhancement aims to improve the quality of user-entered data and prevent errors associated with incorrect INNs.

yozachar commented:
Hi @tabbols95, thanks for the contribution. Country specific validations must go under [`src/validators/i18n/`](https://github.com/python-validators/validators/tree/master/src/validators/i18n)

Please add test cases as well.

tabbols95 commented:
Hi @yozachar, thank you for your feedback. I have moved the changes to the i18n directory and added tests. All tests passed successfully.
Thank you for the library. I use it very actively in my work.

Best regards, Alexander.

yozachar commented on src/validators/i18n/inn.py:
Can you please rename this file as `ru`?

yozachar commented on tests/i18n/test_inn.py:
This then would be `test_ru`.

yozachar commented on src/validators/i18n/__init__.py:
```diff
@@ -5,6 +5,7 @@
 from .fi import fi_business_id, fi_ssn
 from .fr import fr_department, fr_ssn
 from .ind import ind_aadhar, ind_pan
+from .inn import inn as ru_inn
```
You could rename the function as `ru_inn` directly.

tabbols95 commented:
Done

yozachar commented on src/validators/i18n/ru_inn.py:
```diff
@@ -0,0 +1,53 @@
+"""Inn."""
+
+from src.validators.utils import validator
```
Import directly from `validators`, everywhere:

```py
from validators.utils import validator
```

yozachar commented on src/validators/i18n/ru_inn.py:
```diff
@@ -0,0 +1,53 @@
+        (Literal[True]): If `value` is a valid russian individual tax number.
+        (ValidationError): If `value` is an invalid russian individual tax number.
+
+    Returns:
```
Format your files. Double check your docstrings. Refer other modules.
